proposals/dao/swap_dpi.ts

```diff
@@ -0,0 +1,77 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+
+/*
+
+Swap DPI for DAI
+
+Description:
+
+Steps:
+  1 - 
+  2 -
+  3 - 
+
+*/
+
+const fipNumber = 'swap_dpi';
+
+// LBP Swapper config
+const LBP_FREQUENCY = '86400'; // 24 hours
+const MIN_LBP_SIZE = ethers.constants.WeiPerEther.mul(1_000_000); // 1M
+let poolId; // auction pool id
+
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const BalancerLBPSwapperFactory = await ethers.getContractFactory('BalancerLBPSwapper');
+  const dpiToDaiSwapper = await BalancerLBPSwapperFactory.deploy(
+    addresses.core,
+    {
+      _oracle: addresses.chainlinkDpiUsdOracleWrapper,
+      _backupOracle: ethers.constants.AddressZero,
+      _invertOraclePrice: true,
+      _decimalsNormalizer: 0
+    },
+    LBP_FREQUENCY,
+    '100000000000000000', // small weight 10%
+    '900000000000000000', // large weight 90%
```

Comment:
have these weights been run by Storm? would we rather create a sell only PSM than use an LBP?

what are ending weights?

Reply:
honestly I'm more in favor of sell only PSMs at this point too if the goal is liquidation

Reply:
Those weights were premature and WIP, not run by Storm

Reply:
Weights - discussed internally are that 5/95 on a long time horizon are probably okay